src/Symfony/Component/PropertyInfo/Extractor/PhpStanExtractor.php

```diff
@@ -32,9 +32,9 @@
  */
 final class PhpStanExtractor implements PropertyTypeExtractorInterface, ConstructorArgumentTypeExtractorInterface
 {
-    public const PROPERTY = 0;
-    public const ACCESSOR = 1;
-    public const MUTATOR = 2;
+    private const PROPERTY = 0;
+    private const ACCESSOR = 1;
+    private const MUTATOR = 2;
```

Comment:
I don't think that those constants should be public. They're only used internally.